### DISCUSSION TITLE
Data Science(Fundamental Machine Learning)

### DISCUSSION THREAD

**Discussion by hariyantods** [teamalgoritma/community#300] [Q&A] [answered]
Permisi, saya mau nanya, saya ngambil yang Full Stack, di full stack itu akan diajarin Python sampai looping dan function? biasanya setau saya klau Data Science belajar python sampe advance, dan mungkin belajar juga sampai fundamental machine learning, tapi tergantung, yg saya liat klau di sini mengedepankan bahasa pemrograman R, jadi saya pengen tau proporsinya untuk Python, terima kasih.

**Comment by NabiilahArdini** [comment, marked as answer]
Halo @hl9388 ,

Untuk materi spesialisasi Data Analytics di Algoritma (menggunakan Python), scopenya adalah belajar Python untuk keperluan descriptive dan diagnostic analytics. Oleh karena itu materi banyak membahas pemanfaatan library pandas untuk kedua hal tersebut.

Untuk Python basic sendiri, yang akan dipelajari adalah pembuatan object dan tipe-tipe data yang ada. Adapun untuk pembuatan function dapat dibahas pada saat aktivitas Capstone (tugas akhir).

Namun, mentor Algoritma terbuka jika student ingin belajar lebih dari yang diberikan dalam bentuk diskusi di group mentoring yang diadakan setiap minggunya.

Selain itu, course Python maupun R pada Algoritma juga menitikberatkan pada penerapan workflow project yang aplikatif untuk bahasa apapun. Sehingga apabila Anda mempelajari ML menggunakan R, Anda dapat mencari tools Python yang sepadan untuk melakukan tujuan analisis yang sama. Semoga informasi ini dapat bermanfaat.
